feat(workflowengine): Add audio and video MIME types to Automated Tagging dropdown

Ratatou2 opened the pull request:
Fixes: #54462, Related: #30173

## Summary
This PR updates `apps/workflowengine/src/components/Checks/FileMimeType.vue`.
The update enables batch application of MIME types added in PR #54447 through the Automated Tagging dropdown menu.
Without this feature, although various Audio and Video types have been added to MIME types, users still need to add them manually.

## Scope of this PR
This feature simply adds 2 dropdown options to "Automated Tagging" (Audio and Video types).

## How to test
1. Added Audio and Video dropdown types and connected them to existing Audio and Video tags
2. Confirmed that subsequently uploaded Audio and Video files are automatically tagged
(This targets only files that are already included in the existing MIME types in resources/config/mimetypealiases.dist.json)

## Screenshots
### **[UI] Before & After**
<img width="800"  alt="image" src="https://github.com/user-attachments/assets/a3866f33-0327-4675-86ae-5cda50274537" />

---

### **[Code] Before & After**
<img width="800"  alt="image" src="https://github.com/user-attachments/assets/09d349c5-b1a8-4d4d-b59c-f74e849d6527" />

---

### **Test**
**1) Add Tag for Audio & Video**
<img width="800" alt="image" src="https://github.com/user-attachments/assets/fa7ef337-a086-45c2-9104-cf9bb164d5ba" />

**2) Before add files**
<img width="800" alt="image" src="https://github.com/user-attachments/assets/cc811a0f-3560-4f72-8a70-c16df38b5af1" />

**3) File upload and automated tagging working correctly**
<img width="800" alt="image" src="https://github.com/user-attachments/assets/467eb0d6-a591-494d-9e9c-3beb850cf7c6" />

Ratatou2 committed:
feat(workflowengine): Add audio and video MIME types to Automated Tagging dropdown menu

- Add Dropdown Menu for common audio MIME types (audio/mpeg, audio/wav, audio/ogg, etc.)
- Add Dropdown Menu for common video MIME types (video/mp4, video/webm, etc.)
- Improves user experience when creating automated tagging rules for media files

Fixes #54462

Signed-off-by: JinHakChoi <ws423257@gmail.com>

Ratatou2 committed:
build(workflowengine): update dist bundle files
This PR includes the updated build artifacts (dist/workflowengine-workflowengine.js and .js.map) because CI detected modifications and failed without them.

Fixes #54462

Signed-off-by: JinHakChoi <ws423257@gmail.com>

nfebe commented:
Hi @Ratatou2 yeahx unfortunately we do commit the dist folder for the server repo.


But you don't have to push those are we have a build bot that build on PR.

Ratatou2 committed:
build(workflowengine): update dist bundle files (extra files)

Previously, only two dist files were committed.
This commit updates the full dist directory after running a clean build,
since the CI detected additional modifications and failed without them.

Fixes #54462

Signed-off-by: JinHakChoi <ws423257@gmail.com>

blizzz commented:
The other checks also do not run on forks. I invited you to this org, then you can push here into a feature branch; alternatively we could push it to a temporary branch just for CI.

Ratatou2 commented:
> The other checks also do not run on forks. I invited you to this org, then you can push here into a feature branch; alternatively we could push it to a temporary branch just for CI.

Thank you very much for the invitation to the org 🙏
This will definitely make future contributions and CI runs much smoother.
Once I’m added to the org, I’ll push my feat/... branch here! : )
